### ISSUE TITLE
Android - INJI app is not supporting face authentication 

### ISSUE THREAD

**Issue by santhoshsunder** [mosip/inji#518]
**Describe the bug**
INJI app is not supporting face authentication in bio metrics unlock 

**To Reproduce**
Prerequiste : Remove all the fingure prints added in the device and add face authentication

1. Authenticate app and enter it
2. Go to profile section
3. Click on "Biometric unlock" toggle button

**Expected behavior**
App should be capable with face authentication as one of the authentication method

**Observed**
App is allowing face authentication as bio metrics, its projecting error message

**Screenshots**
![Screenshot_2023-01-31-15-22-31-650_io mosip residentapp](https://user-images.githubusercontent.com/102220709/215730731-d96e33af-b82a-48b0-abb7-2580948403f0.jpg)

**Smartphone (please complete the following information):**
- Device: Redmi Note 10 Lite
- OS: Android 10
- MOSIP Version: 1.2.0.1
- MOSIP server: qa-1201-b2.net
- APP link; https://github.com/mosip/inji/actions/runs/3986424542

**Comment by rakhimosip** [comment]
Yes, in devices that support the face unlock feature, this is expected to work.

**Comment by jannahadlaon** [comment]
Tested on: _io.mosip.residentapp-0.4.1-rc8-2-6c7fbe0-fix_375-optimize-ios-sharing-newlogic_20230127_2003_universal.apk_

**WORKS AS EXPECTED**
I was not able to replicate the issue above
@pmigueld @kyanthony @santhoshsunder @Sujithbn 

**Device**
Samsung Galaxy A23 -  Android 12

**Screenshots**

https://user-images.githubusercontent.com/102940764/216017713-3812d859-2e67-4beb-9606-85ccc07e8fd3.mov

**Comment by santhoshsunder** [comment]
we tested this issue in four devices which all are capable of face authentication, it worked only in one device, i have added the details of working device and not working device details bellow, and attaching the video of this issue

**VIDEO** 
https://user-images.githubusercontent.com/102220709/216039749-5eea2f1d-b2e2-4868-9d7e-46a79a6176b4.mp4

**Working device :**
Android version : 11
BLE version : 5.2
device name : One Plus Nord 2

**Not working devices :**
Android version : 12
BLE version : 5.0
device name : vivo Y73

Android version : 11
BLE version : 5.0
device name : Redmi K20 Pro

Android version : 10
BLE version : 5.0
device name : redmi note 10 lite 

Remaining details :
MOSIP Version: 1.2.0.1
MOSIP server: qa-1201-b2.net
APP link; https://github.com/mosip/inji/actions/runs/4062244892

**Comment by pmigueld** [comment]
@santhoshsunder For all the devices where the face biometrics did not work, can you check the behavior in other apps on the same device (e.g. does the another app allow face unlock but not Inji?)

I was able to replicate on an Infinix Hot 11 2022 (Android 11) and it only recognizes fingerprint biometrics **in apps**. It has face biometrics but only uses it to unlock the phone. I temporarily removed the capability check to force using biometrics unlock with only face data and not fingerprint and instead it showed me the pattern unlock for the app. It seems not all face biometrics in Android is usable in apps.

cc: @jannahadlaon @kyanthony @rakhimosip

**Comment by Sujithbn** [comment]
This is a limitation in Android and Device level will not be able to address this on Beta release.

**Comment by Sujithbn** [comment]
As discussed with Rounak, priority has been reduced for this defect
